test/cppsuite/tests/background_compact.cpp

```diff
@@ -300,12 +300,9 @@ class background_compact : public test {
         logger::log_msg(
           LOG_INFO, type_string(tw->type) + " thread {" + std::to_string(tw->id) + "} commencing.");
 
-        bool enabled = false;
-
         while (tw->running()) {
-            enabled = !enabled;
 
```

Comment:
Can you add a comment here saying we are doing this on purpose?

Reply:
I've added
```
    /*
    * The API supports enabling or disabling the background compact server multiple times
    * in a row. Randomly pick whether we are enabling or disabling to cover all state
    * changes.
    */
```